unstructured_inference/models/tables.py

```diff
@@ -161,12 +161,17 @@ def run_prediction(
         self,
         x: Image,
         pad_for_structure_detection: int = inference_config.TABLE_IMAGE_BACKGROUND_PAD,
+        ocr_tokens: List = [],
     ):
         """Predict table structure"""
         outputs_structure = self.get_structure(x, pad_for_structure_detection)
-        tokens = self.get_tokens(x=x)
+        if ocr_tokens == []:
```

Comment:
probably should allow it to be empty if the OCR detects no text. And we can set the default "not OCR'ed" state as `None`.
Additionally we may want to specify a bit more of the requirement for `ocr_tokens` is a list of dictionaries. For now we can just add the docstring to layout what is required for each token. And leave a note for future FIXME to refactor token data into a dataclass so we have clear expectations on what it should "look" like.